boards/xtensa/m5stack_stamps3/doc/index.rst

```diff
@@ -0,0 +1,199 @@
+.. _m5stack_stamps3:
+
+M5Stack StampS3
+###############
+
+Overview
+********
+
+M5Stack StampS3 is an ESP32-based development board from M5Stack.
+It features the following integrated components:
+
+- ESP32-S3FN8 chip (240MHz dual core)
+- 512KB SRAM
+- 384KB ROM
+- 8MB Flash
+- Wi-Fi
+- Bluetooth
+- User-Button
+
+.. figure:: img/m5stack_stamps3.webp
+        :align: center
+        :alt: M5Stack-StampS3
```

Comment:
be consistent with (official) spelling: (please perform global search/replace)

```suggestion
        :alt: M5Stack StampS3
```

Reply:
Fixed in [38f0a59](https://github.com/zephyrproject-rtos/zephyr/commit/38f0a59c9701f755f1b41c0489c642aeb2e0bc7b)